website/docs/docs/collaborate/explore-projects.md

```diff
@@ -162,8 +162,41 @@ Under the the **Models** option, you can filter on model properties (access or m
 
 </Expandable>
 
+<Expandable alt_header="Trust signals for models" lifecycle="preview">
+
+Trust signal icons offer a quick, at-a-glance view of data health when browsing your models in dbt Explorer. These icons keep you informed on the status of your model's health using the indicators **Healthy**, **Caution**, **Degraded**, and **Unknown**. For accurate health data, ensure the resource is up-to-date and has had a recent job run.
+
+Each trust signal icon reflects key data health components, such as whether tests were run successfully, whether the resource has a description, whether it has been built in the past 30 days, and more. 
+
+Access trust signals using the search function or when clicking on **Models** under the **Resource** tab. Hover over or click the trust signal to see detailed information.
+
+#### Healthy 
+- Resource is healthy and ran all tests successfully
+- Successfully built in its last run
+- Has a description
+- Has tests configured
+- Has been built in the past 30 days 
+<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-healthy.png" title="Healthy trust signal icon"/>
+
+#### Caution
+- Successfully built in its last run
+- Is missing a description
+- Does not have any tests configured
+- Has been built in the past 30 day
+<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-caution.png" title="Caution trust signal icon"/>
```

Comment:
We might want to get product to update the text in the screenshots because this is really confusing when it says the resource is "caution" because it successfully ran on it's last run. Are these fields dynamic - when it's caution does it change the fields that put it in caution only or does it display the same text all the time?